src/frontend/src/handler/create_mv.rs

```diff
@@ -76,7 +76,7 @@ pub fn gen_create_mv_plan(
 
     let (database_id, schema_id) = session.get_database_and_schema_id_for_create(schema_name)?;
 
-    let definition = query.to_string();
```

Comment:
With this previous method, `WITH` options cannot be preserved. That'll be problematic for sinks.